cvmfs/mountpoint.cc

```diff
@@ -1167,6 +1167,8 @@ MountPoint *MountPoint::Create(
   mountpoint->CreateTables();
   mountpoint->SetupBehavior();
 
+  mountpoint->magic_xattr_mgr_ = new MagicXattrManager(mountpoint.weak_ref());
```

Comment:
Let's put that in `SetupBehavior`.  The `hide_magic_xattrs_` flag can be incorporated in the `MagicXattrManager`